docs/start-dev-environment.md

```diff
@@ -19,24 +19,13 @@ cp .env-back.example .env-back
 ```
 Make modifications if required.
 
-### 3a. Build and launch
+### 3. Build and launch
 
 ```bash
 docker-compose build
 docker-compose run web rake db:create
 docker-compose run web rake db:reset
 docker-compose up
-```
-
-### 3b. Alternative: Run the front-end locally
-The previous command will run all application components in Docker, including front-end. If you want to actively develop on Mac or Windows, that can be a bit slow. 
-
-If you rather run front locally on your machine instead of in Docker, use these commands instead.
-
-```bash
-docker-compose run web rake db:create
-docker-compose run web rake db:reset
-docker-compose run --service-ports web
 cd front
```

Comment:
Maybe doing this in a separate code block would be useful, and adding that this should happen in a new terminal window?

Reply:
Good point, done !